.factory/automation.yml

```diff
@@ -97,9 +97,9 @@ build:
         export ARTIFACT_PASSWORD=$REPO_VATICLE_PASSWORD
         bazel run @vaticle_dependencies//tool/bazelinstall:remote_cache_setup.sh
         bazel run @vaticle_dependencies//distribution/artifact:create-netrc
-        tools/start-core-server.sh
-        bazel test //tests --test_arg=-- --test_arg=behaviour::concept --test_arg=--test-threads=1 --test_output=streamed && export TEST_SUCCESS=0 || export TEST_SUCCESS=1
-        tools/stop-core-server.sh
+        source tools/start-cluster-servers.sh # use source to receive export vars
+        bazel test //tests --test_env=ROOT_CA=$ROOT_CA --test_arg=-- --test_arg=behaviour::concept --test_arg=--test-threads=1 --test_output=streamed && export TEST_SUCCESS=0 || export TEST_SUCCESS=1
+        tools/stop-cluster-servers.sh
```

Comment:
we should make two sets of tests in `automation.yml`: one for cluster (`test-behaviour-concept-cluster`) and (`test-behaviour-concept-core`)

Same pattern for `behaviour-typeql` below

Reply:
We decided that it's out of the scope now. https://github.com/vaticle/typedb-client-rust/issues/52